seatunnel-apis/seatunnel-api-flink/src/main/java/org/apache/seatunnel/flink/util/SchemaUtil.java

```diff
@@ -117,8 +120,27 @@ public static FormatDescriptor setFormat(FormatType format, Config config) throw
         return formatDescriptor;
     }
 
-    private static void getJsonSchema(Schema schema, ObjectNode json) {
-        Iterator<Map.Entry<String, JsonNode>> nodeIterator = json.fields();
+    private static void getJsonSchema(Schema schema, String json) {
```

Comment:
Sorry, but I don't think this method can implement schema parsing in list form. Although the fields in the list are processed cyclically, the returned schema does not reflect the attributes of the list. Maybe a field can be added for this situation: `list`, which points to the list at the outermost level of the schema

Reply:
Yes, there is a problem. Let me fix it

Reply:
The Flink schema is not familiar enough to know if this is the right thing to do
![image](https://user-images.githubusercontent.com/43572770/183120889-0a3fa3ea-e9ea-45d1-a4ff-334eac8f27f5.png)

Reply:
@Hisoka-X

Reply:
Sorry for late reply, you are right, same as flink.